dbms/src/Common/tests/gtest_magic_enum.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <common/crc64.h>
+#include <gtest/gtest.h>
+
+#include <magic_enum.hpp>
+using namespace magic_enum;
```

Comment:
```suggestion
```

Reply:
Nice one! I'll consider to add more tests using api which may replace existing enum related code.

Reply:
I evaluated the enum usage in existing projects and added the magic API test cases for accessing and converting enum types, I can add some more advanced APIs in the future if needed.